massa-versioning/src/versioning.rs

```diff
@@ -921,12 +922,14 @@ impl MipStoreRaw {
                 .stats
                 .network_version_counters
                 .get(&mi.version)
-                .unwrap_or(&0) as f32;
-            let block_count_considered = self.stats.config.block_count_considered as f32;
-
-            let vote_ratio_ = 100.0 * network_version_count / block_count_considered;
-
-            let vote_ratio = Amount::const_init(vote_ratio_.round() as u64, 0);
+                .unwrap_or(&0); // as f32;
+
+            let vote_ratio = Ratio::new(100, 1);
```

Comment:
we should not be multiplying everything by 100 here, it is already a ratio

Reply:
fixed